recipes-browser/packagegroups/packagegroup-wpewebkit-depends.bb

```diff
@@ -33,7 +33,7 @@ RDEPENDS:packagegroup-wpewebkit-depends = "\
 
 RDEPENDS:packagegroup-wpewebkit-depends-sys-extended = "\
     curl \
-    ${@bb.utils.contains_any('LAYERSERIES_CORENAMES', 'dunfell', 'dhcp-client', 'dhcpcd', d)} \
+    dhcpcd \
```

Comment:
If this is only for getting an IP address, perhaps nowadays we can do with systemd network facilities?
(just an aside comment, I don't think that it's necessary to complicate this change --which is a good clean-up-- because of this)